.gitignore

```diff
@@ -1 +1,2 @@
-dist/
\ No newline at end of file
+dist/
+.idea/
```

Comment:
I know this is somewhat common, but I think it is wrong to put rules specific to your OS or IDE into a project repo; they should go into `~/.gitignore` instead, so they apply to all the repos on your machine. E.g. on a macOS machine using JetBrains IDE:

```console
$ cat ~/.gitignore
.backup~/
.idea/

# OS generated files #
.DS_Store
ehthumbs.db
Icon?
Thumbs.db
.fseventsd/
```

I know @Nino-K will just approve and merge anyways, but this way you can fix the issue once on your machine and never have to deal with it again (if you share dotfiles across your machines).

Reply:
Okay, that's fine, too! I'll remove the commit.